plasma_framework/contracts/src/framework/Protocol.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.5.0;
+
+library Protocol {
+    uint8 constant MVP_VALUE = 1;
```

Comment:
yeah, actually I started with `enum`. (this commit: https://github.com/omisego/plasma-contracts/pull/237/commits/ea6edabc8ed644056d9a073cb0a76e1fcb784975)

But it seems like solidity cannot enum with custom value, so it would be started with 0. This would lead to confusion with current implementation of registry since if the protocol is not registered, it would return 0 as well (default value from mapping). As a result, I would like to avoid any value collide with empty value.

well....alternative is that we have enum with three value, with the first one being like `DO_NOT_USE_ME` lol?

Reply:
Oh that's awful! 😝 
I prefer the way you've implemented it